app/components/Search/index.js

```diff
@@ -71,7 +74,7 @@ const SuggestWrapper = styled.div`
   max-width: 620px;
   width: 100%;
   top: calc(100% - 11px);
-  z-index: -1;
+  z-index: 1;
```

Comment:
Does this change also work well on mobile?

Reply:
Was there a specific reason why this was -1? It seems a bit arbitrary, and will appear underneath content in (search history items, in this case)

It works in the browser and on mobile (tested on an old iphone 5)

I found an existing bug on mobile: clicking a search result doesn't do anything.
I'll add it to the backlog.

Reply:
If I remember correctly, the negative z-index was to make sure it also displayed correctly in IE11

Reply:
Looks like it's working in IE11 for z-index:1